.taskcluster.yml

```diff
@@ -88,9 +94,11 @@ tasks:
       github:
         events:
           - tag
+    scopes:
+      - "secrets:get:project/application-services/publish"
     payload:
-      maxRunTime: 3600
-      deadline: "{{ '2 hours' | $fromNow }}"
+      maxRunTime: 7200
+      deadline: "{{ '4 hours' | $fromNow }}"
```

Comment:
Yikes, 4 hour build!

Reply:
Actually a 2 hour build (`maxRunTime`).  I think `deadline` is the sum of the time to schedule the job in TC and the time to actually complete, although I can't guarantee that.